src/components/bookList.js

```diff
@@ -1,20 +1,54 @@
 import React from 'react';
 import Book from './book';
+import Form from './form';
 
 const BooksList = () => {
-  const bookList = [
-    {
-      id: 1,
-    },
-    {
-      id: 2,
-    },
-  ];
-  const list = bookList.map((item) => (<li key={item.id}><Book /></li>));
+  const state = {
+    bookList: [
+      {
+        id: 1,
+        title: 'The way of Kings',
+        author: 'Brandon',
+      },
+      {
+        id: 2,
+        title: 'The dragon reborn',
+        author: 'Marcus Rufus',
+      },
+    ],
+  };
+
   return (
-    <ul>
-      {list}
-    </ul>
+  // const list = bookList.map((books) => <li key={books.id}><Book books={books} /></li>);
```

Comment:
[REQUIRED] It would be nice if you removed unused code for it makes your code base look untidy